web/components/contract/contract-tabs.tsx

```diff
@@ -364,54 +365,66 @@ export const CommentsTabContent = memo(function CommentsTabContent(props: {
           }
         />
       )}
-      {parentComments.slice(0, parentCommentsToRender).map((parent) => (
-        <FeedCommentThread
-          key={parent.id}
-          contract={contract}
-          parentComment={parent}
-          threadComments={commentsByParent[parent.id] ?? []}
-          trackingLocation={'contract page'}
-          idInUrl={idToHighlight}
-          showReplies={
-            !isBountiedQuestion || (!!user && user.id === contract.creatorId)
-          }
-          childrenBountyTotal={
-            contract.outcomeType == 'BOUNTIED_QUESTION'
-              ? childrensBounties[parent.id]
-              : undefined
-          }
-          onSubmitReply={loadNewer}
-          bets={bets?.filter(
-            (b) =>
-              b.replyToCommentId &&
-              [parent]
-                .concat(commentsByParent[parent.id] ?? [])
-                .map((c) => c.id)
-                .includes(b.replyToCommentId)
-          )}
-        />
-      ))}
 
-      <div className="relative w-full">
-        <VisibilityObserver
-          onVisibilityUpdated={onVisibilityUpdated}
-          className="pointer-events-none absolute bottom-0 h-[75vh]"
-        />
-      </div>
-
-      {!user && (
-        <Button
-          onClick={withTracking(
-            firebaseLogin,
-            'sign up to comment button click'
-          )}
-          className={clsx('mt-4', comments.length > 0 && 'ml-12')}
-          size="lg"
-          color="gradient"
-        >
-          Sign up to comment <ArrowRightIcon className="ml-2 h-4 w-4" />
-        </Button>
-      )}
```

Comment:
Uh, I don't think you want to delete this code?

Reply:
lol good catch, have re-added it to my current code.